posts/mpox-preparedness/index.qmd

```diff
@@ -0,0 +1,100 @@
+
+### Forecasting and nowcasting infection dynamics
+
+::: {.callout .callout-info}
+Forecasting and nowcasting of infections are crucial for planning and resource allocation during an outbreak. Forecasting is the prediction of future cases, deaths, or other outcomes, while nowcasting is the prediction of the current outbreak situation. These predictions can help public health authorities to anticipate the trajectory of the outbreak and to implement timely interventions.
+:::
+
+[`{EpiNow2}`](https://epiforecasts.io/EpiNow2/) and [`{epinowcast}`](https://package.epinowcast.org/) provide functions to forecast and nowcast the number of cases. The data required for `{EpiNow2}` have already been described in the previous section. The `{epinowcast}` package similarly requires data on the number of cases reported per date. `{epinowcast}` does not currently support forecasting but there are plans to add this functionality in future versions.
+
+### Estimating disease severity
+
+::: {.callout .callout-info}
+The case fatality ratio (CFR) is often used to assess the severity of a disease. CFR here refers to the proportion of deaths among confirmed cases.
+:::
+
+With incidence data on the number of cases reported and the number of deaths reported, the [`{cfr}`](https://epiverse-trace.github.io/cfr/) package can be used to estimate the case fatality rate and its uncertainty. Importantly, it accounts for the delay between the onset of symptoms and death, which is crucial for accurate estimation of the case fatality rate.
+
+Here again, `{EpiNow2}` can be used to estimate the time-varying case fatality ratio using the same data as for the reproduction number. `{EpiNow2}` can estimate other severity metrics, such as the case hospitalisation ratio, given data on cases and hospitalisations, and the hospitalisation fatality ratio, if data on hospitalisations and associated deaths are available.
+
+### Assessing the impact of interventions
+
+::: {.callout .callout-info}
+mpox can be mitigated with behaviour change, treatment, and vaccination. Here, a few tools are available to assess the impact of intervention scenarios.
+:::
+
+[`{epidemics}`](https://epiverse-trace.github.io/epidemics/) provides ready compartmental models to estimate the impact of vaccination and non-pharmaceutical interventions like behaviour change, which can conceptually be modelled as a reduction in the transmission rate through changes in the population contact structure.
+
+If we want to explore population-level outbreak dynamics, `{epidemics}` allows for stratifying the population into arbitrary groups, specifying the contact structure between these groups, and rates of interventions. The data required to run these models include: population structure, contact structure, and timing and magnitude of interventions.
```

Comment:
finalsize can also be used for basic scenario modelling. Should it be added here as well?

Reply:
Think we should be careful about suggesting {finalsize} for scenarios given limited transmissibility of mpox in general population (as `finalsize()` will return outbreak size = 0 if R0<1). But the `r_eff()` function could be useful to understand how heterogenity in contacts influences $R$ in subpopulations (like the Endo et al Science paper)

Reply:
I think I'll leave it out.